### PR TITLE
Adding Podium code style

### PR DESCRIPTION
**Pull request by leftieFriele**
The purpose of this PR is to add the code style used in the rest of Podium into this module.
- Adds Eslint configs from podlet module
- Updates `.prettierrc` with content from podlet module
- Adds run targets `lint` and `lint:fix`

When reviewing you probably don't want to look at all the source files 😅 , the gist of the PR is in:
- [.eslintrc](https://github.com/podium-lib/podlet-server/blob/code-style/.eslintrc)
- [.prettierrc](https://github.com/podium-lib/podlet-server/blob/code-style/.prettierrc.prettierrc)

ℹ️  The pipeline is not updated to use this yet, so this won't break anything if merged.

### CODE REVIEW

**Review by j4hr3n — COMMENTED** [review]
Looks good to me 👍

**Comment by leftieFriele** [comment]
There are still some issues left, but I'm not sure we actually want to fix them and rather tweak the rule set:
```
21:9  error  for..in loops iterate over the entire prototype chain, 
which is virtually never what you want. Use Object.{keys,values,entries}, 
and iterate over the resulting array  no-restricted-syntax
```
```
16:5  error  iterators/generators require regenerator-runtime, 
which is too heavyweight for this guide to allow them. 
Separately, loops should be avoided in favor of array iterations  no-restricted-syntax
```
```
21:9  error  The body of a for-in should be wrapped in an if statement to 
filter unwanted properties from the prototype
```

You can see this when running `npm run lint` on this branch.

File: lib/pino-dev-transport.js [hunk 1]
```diff
@@ -5,11 +5,11 @@ import chalk from "chalk";
 const coloriseStatusCode = (statusCode) => {
   if (statusCode >= 500) {
     return chalk.red(statusCode);
-  } else if (statusCode >= 400) {
+  } if (statusCode >= 400) {
```
[the remark this answers is not in the document]
Comment (leftieFriele) [reply lib/pino-dev-transport.js]: 🤔 I'm note sure how a switch would work here, given that we're comparing `>= 400` etc. the linter removed the `else` part of the if's as part of some rule I assume, wether it's better now could be up for debate 😅

File: commands/i18n.js [hunk 2]
```diff
@@ -1,42 +1,41 @@
...
-    describe: "i18n command to run",
+    describe: 'i18n command to run',
   });
 
   return yargs.argv;
 };
 
 export const handler = async (argv) => {
-  const { command } = argv;
+  const { command: commandArgument } = argv;
```
Comment (trygve-lie) [inline commands/i18n.js:30]: Why is it changing this?

**Comment by trygve-lie** [comment]
> 21:9  error  for..in loops iterate over the entire prototype chain, 
which is virtually never what you want. Use Object.{keys,values,entries}, 
and iterate over the resulting array  no-restricted-syntax

I think we should "fix" this one.

File: commands/i18n.js (hunk 2 above)
Comment (leftieFriele) [reply commands/i18n.js:30]: It collided with the `command` declared in line 8 in the same file.

**Comment by digitalsadhu** [comment]
> 16:5 error iterators/generators require regenerator-runtime,
which is too heavyweight for this guide to allow them.
Separately, loops should be avoided in favor of array iterations no-restricted-syntax

Yea I'm not a fan of this rule.

**Review by digitalsadhu — APPROVED** [review]

**Review by trygve-lie — APPROVED** [review]
lgtm :+1:

**Pull request merged**